components/staging/staging.js

```diff
@@ -325,6 +335,8 @@ FileViewModel.prototype.setState = function(state) {
   this.isNew(state.isNew);
   this.removed(state.removed);
   this.conflict(state.conflict);
+  this.additions(state.additions);
+  this.deletions(state.deletions);
```

Comment:
instead of `this.deletionsFormatted` or `this.additionsFormatted`, I think you could do something like`this.additions(state.additions ? '+' + state.additions: '')`;